lib/serve.js

```diff
@@ -51,6 +52,15 @@ const serve = (watchPath, options) => new Promise((resolve, reject) => {
 
           form.on('file', (field, file) => {
             files.push([field, file]);
+            var data = {
+              'aframe': {
```

Comment:
That's why I asked about `deepAssign`.  I figured that it would be smart about that.   I'll test though since the data structure would allow for multiple video assets.